components/productions/index/event.vue

```diff
@@ -6,6 +6,7 @@
         :name="name"
         :checked="isSelected"
         :disabled="isDisabled"
+        :aria-label="name"
```

Comment:
I think `pub-checkbox` uses `<label>` underneath, so maybe we don't even need this? If we do, maybe we should add it to the component so it sets it automatically for every checkbox based on the `:name` so we don't need to duplicate it everywhere?